account_tax_unece/models/account_tax.py

```diff
@@ -27,3 +27,15 @@ class AccountTax(models.Model):
     unece_categ_code = fields.Char(
         related='unece_categ_id.code', store=True,
         string='UNECE Category Code')
+    unece_due_date_id = fields.Many2one(
+        'unece.code.list', string='UNECE Due Date',
+        domain=[('type', '=', 'date')], ondelete='restrict',
+        help="Select the due date of that tax from the official "
+        "nomenclature of the United Nations Economic "
```

Comment:
Align this at the level of the first `"`

Reply:
flake8 is happy with it. I think it's a matter of taste here...